### PR TITLE
add isPrevUserPresent

### PR DESCRIPTION
**Pull request by milosst94**

### CODE REVIEW

File: src/app/standup/standup.controller.ts [hunk 1]
```diff
@@ -34,14 +37,16 @@ export class StandupController {
 
   @ApiBearerAuth()
   @UseGuards(AuthGuard('jwt'))
-  @Post('/:workspaceId/finish-standup')
+  @Put('/:workspaceId/finish-standup')
   async finishStandup(
     @Body() dto: StandupDto,
```
Comment (Branimir) [inline src/app/standup/standup.controller.ts]: This looks like not needed.

File: src/app/standup/standup.service.ts
Comment (Branimir) [inline src/app/standup/standup.service.ts:1]: We should not allow duplicates in absentUsers and attendees arrays. 
Also, when an user is added initially as attendee, and then as absent, we should remove that user from attendees and add to the absentUsers array.

**Review by Branimir — COMMENTED** [review]
Please check the comments

**Review by Branimir — APPROVED** [review]

**Review by stefanj122 — APPROVED** [review]
Brutalno

**Pull request merged**